entity-framework/core/miscellaneous/cli/powershell.md

```diff
@@ -143,8 +146,8 @@ Parameters:
 
 | Parameter                         | Description                                                                                                             |
 |:----------------------------------|:------------------------------------------------------------------------------------------------------------------------|
-| <nobr>-Name \<String><nobr>       | The name of the migration. This is a positional parameter and is required.                                              |
-| <nobr>-OutputDir \<String></nobr> | The directory (and sub-namespace) to use. Paths are relative to the target project directory. Defaults to "Migrations". |
+| -Name \<String>                   | The name of the migration. This is a positional parameter and is required.                                              |
```

Comment:
Wondering if adding the parameters inside backticks could help with the rendering problem or make it worse.

```suggestion
| `-Name \<String>`                 | The name of the migration. This is a positional parameter and is required.                                              |
```

Reply:
Should probably remove the backslash in front of String if you're wrapping in backticks

Reply:
FWIW I played around with this a bit... the backticks make the text stand out much more and is quite unnatural/jarring to me. Making the browser very narrow makes the second argument drop to a separate line (i.e. `-Name\n<string>`) but it seem really OK to me (all very subjective of course).

Unless someone objects (@bricelam?) I'd say we merge as-is (no backticks, no special non-breaking directive).

Reply:
Indeed. Fixed suggestion.

Reply:
It's up to you guys. On the contributor guide they also talk about some new table formats but I've never managed to use them and be successful on solving my rendering issues with them
But leaving here for FYI: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#tables